src/Hangfire.Core/Dashboard/Pages/FailedJobsPage.cshtml

```diff
@@ -7,13 +7,34 @@
     Layout = new LayoutPage("Failed Jobs");
 
     int from, perPage;
+    string filterString = Query("filterString");
+    string startDate = Query("startDate");
+    string endDate = Query("endDate");
 
     int.TryParse(Query("from"), out from);
     int.TryParse(Query("count"), out perPage);
 
     var monitor = Storage.GetMonitoringApi();
-    var pager = new Pager(from, perPage, monitor.FailedCount());
-    var failedJobs = monitor.FailedJobs(pager.FromRecord, pager.RecordsPerPage);
+    Hangfire.Dashboard.Pager pager;
+    Hangfire.Storage.Monitoring.JobList<Hangfire.Storage.Monitoring.FailedJobDto> failedJobs;
+
+    if (!string.IsNullOrEmpty(filterString) && !string.IsNullOrEmpty(startDate) && !string.IsNullOrEmpty(endDate) ) {
```

Comment:
use flat style braces please as mentioned before.  should be consistent like the whole file.